nekoyume/Assets/_Scripts/State/LocalLayerModifier.cs

```diff
@@ -280,7 +264,7 @@ long blockIndex
         public static void RemoveNewAttachmentMail(
             Address avatarAddress,
             Guid mailId,
-            bool resetState = true)
+            bool resetState = default)
```

Comment:
I know the bool default value is false, is this okay?

Reply:
Yes it is. I have fixed all other arguments.